src/components/ui/Countdown.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+import { useState, useEffect } from "react";
+
+interface CountdownState {
+    days: number;
+    hours: number;
+    minutes: number;
+    seconds: number;
+  }
+
+  const useCountdown = (targetDate: Date): CountdownState => {
+    const [countdown, setCountdown] = useState<CountdownState>({
+      days: 0,
+      hours: 0,
+      minutes: 0,
+      seconds: 0,
+    });
+  
+    useEffect(() => {
+      const intervalId = setInterval(() => {
+        const now = new Date();
+        const delta = targetDate.getTime() - now.getTime();
+  
+        if (delta < 0) {
+          clearInterval(intervalId);
+          return;
+        }
+  
+        const days = Math.floor(delta / (1000 * 60 * 60 * 24));
+        const hours = Math.floor((delta % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+        const minutes = Math.floor((delta % (1000 * 60 * 60)) / (1000 * 60));
+        const seconds = Math.floor((delta % (1000 * 60)) / 1000);
+  
+        setCountdown({ days, hours, minutes, seconds });
+      }, 1000);
+  
+      return () => clearInterval(intervalId);
+    }, [targetDate]);
+  
+    return countdown;
+  };
+
+const Countdown = () => {
+    const targetDate = new Date('2024-06-30T23:59:59Z')
```

Comment:
Let's do the real Axios get to https://plankton-app-lht9q.ondigitalocean.app/schedule and assume the first element in the array.  Ill get the API to sort by date.

```json
[
    {
        "_id": "666d11c67723075b413142d6",
        "title": "The 'Mainnet' Test Event",
        "date": "2025-01-01T01:00:00.000Z",
        "location": "Eatons Hill, Brisbane",
        "description": "This is the first event of the Bitcoin Poker tour.  Real Bitcoin is used for this event.",
        "game_type": "No Limit Texas Holdem",
        "buy_in": 0.001,
        "start_stack": 20000,
        "blind_levels": 20,
        "__v": 0
    }
]
```